index.js

```diff
@@ -482,7 +514,6 @@ module.exports = {
         // look for new epochs that we're added to
         pull(
           ssb.db.query(
-            // TODO: does this output new stuff if we accept an invite to an old epoch and then find additions to newer epochs?
```

Comment:
answer seems to be no, but it's fine since acceptInvite now consumes all epochs it can find we've been invited to